pkgs/development/python-modules/pyarlo/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, sseclient-py
+}:
+
+buildPythonPackage rec {
+  pname = "pyarlo";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03zjydi7l8flslggasydv5ychfwagfjavcw892mpr97rjka5aqns";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    sseclient-py
+  ];
+
+  # tests are not present in the PyPI source
```

Comment:
That's a good enough reason fetch from GitHub instead. It gives us more confidence that the packaging actually works, especially when we need to do some patching, or the package gets automatically updated by e.g. @r-ryantm.

Reply:
Source is now pulled from GitHub and tests are executed.